.pre-commit-config.yaml

```diff
@@ -18,11 +18,11 @@ repos:
 #     hooks:
 #       - id: clang-format
   - repo: https://github.com/codespell-project/codespell
-    rev: v2.1.0
+    rev: v2.2.1
     hooks:
       - id: codespell
   - repo: https://github.com/crate-ci/typos
-    rev: v1.11.0
+    rev: typos-dict-v0.8.8
```

Comment:
this one seems incorrect

Reply:
Looks like only a [tagging problem](https://github.com/crate-ci/typos/issues/390)

Reply:
@egecetin so should we close this PR and wait for the next one? 
I'm not sure what to do with this 🤔

Reply:
Actually we should merge it since it is a really update. Because they have two different tags (vX.X.X and typos-dict-vY.Y.Y) for same code. I really don't understand why they use two different tags for same thing

Reply:
I'm not a pre-commit expert, but is there a way it can only look at releases and not at tags?
The releases are a bit more consistent...:
https://github.com/crate-ci/typos/releases

Reply:
Currently I don't have much time but will check it

Reply:
Apparently someone already raised this issue with `typos`: https://github.com/pre-commit/pre-commit/issues/2366

Unfortunately there is no way around it 😕 

But @egecetin as you mentioned, this is only a tagging issue so probably safe to merge